src/server/server.cc

```diff
@@ -39,9 +40,15 @@ void Server::run()
 
     INFO("Server Started");
 
+    // The rules configure the server
+    rules_config(&config_);
+    NOTICE("Game configuration:");
+    NOTICE("- name: %s", config_.name);
+    NOTICE("- %d players", config_.player_count);
```

Comment:
Would it be annoying to also rename all the *player* variables that are actually representing *clients* into, well, *clients*? `client_count`, `ClientType`, `client_type`, `client_id`, etc.

Reply:
Yes we should. I've added a TODO to refactor rules::Player into rules::Client and will address that in another PR.